diff --git a/kernel/Kconfig.hz b/kernel/Kconfig.hz
index 38ef6d068..7216e9c70 100644
--- a/kernel/Kconfig.hz
+++ b/kernel/Kconfig.hz
@@ -17,6 +17,12 @@ choice
 	 per second.
 
 
+	config HZ_12
+		bool "12 HZ"
+	help
+	  12 Hz is for very low end device, like my emulator writen in
+	  python.
+
 	config HZ_100
 		bool "100 HZ"
 	help
@@ -50,6 +56,7 @@ endchoice
 
 config HZ
 	int
+	default 12 if HZ_12
 	default 100 if HZ_100
 	default 250 if HZ_250
 	default 300 if HZ_300
